test/examples_test.go

```diff
@@ -43,14 +43,41 @@ func TestErrorHandlingOnChain(t *testing.T) {
 		Run("--grep", "Local")
 }
 
+func TestErrorHandlingCombined(t *testing.T) {
+	errorHandlingName := "error_handling_attest_fn_call"
+	errorHandlingDir := filepath.Join(examplesDir, errorHandlingName)
+	errorHandlingOnChainName := "error_handling_on_chain"
+	errorHandlingOnChainDir := filepath.Join(examplesDir, errorHandlingOnChainName)
+	saveDir := errorHandlingOnChainDir + "/tmp"
+
+	t.Run(errorHandlingName, func(t *testing.T) {
+		NewTestscriptTest(t, errorHandlingDir, saveDir).
```

Comment:
Here we run the `error_handling_attest_fn_call` examples test but give it a save directory of `error_handling_on_chain/tmp/`. So, when it uses the `saveFile` command it will write files to the error handling on-chain example directory